pkg/apis/core/v1alpha1/types.go

```diff
@@ -44,6 +44,8 @@ type DataVolumeSpec struct {
 	PVC *corev1.PersistentVolumeClaimSpec `json:"pvc"`
 	//DataVolumeContentType options: "kubevirt", "archive"
 	ContentType DataVolumeContentType `json:"contentType,omitempty"`
+	//VolumeMode indicates whether the PV is a block PV or not
+	VolumeMode corev1.PersistentVolumeMode `json:"volumeMode,omitempty"`
```

Comment:
So I don't think this is needed at all. The PersistentVolumeClaimSpec already has a VolumeMode element, so we can already define a DataVolume with block mode. I just verified it is not used in any code at all. everything uses the VolumeMode of the PVC.